client/dist/styles.css

```diff
@@ -0,0 +1,125 @@
+@font-face {
+font-family: 'Airbnb Cereal App Book';
+font-style: normal;
+font-weight: normal;
+src: local('Airbnb Cereal App Book'), url('./fonts/airbnb-cereal-app-cufonfonts-woff/AirbnbCerealBook.woff') format('woff');
+}
+
+
+@font-face {
+font-family: 'Airbnb Cereal App Light';
+font-style: normal;
+font-weight: normal;
+src: local('Airbnb Cereal App Light'), url('./fonts/airbnb-cereal-app-cufonfonts-woff/AirbnbCerealLight.woff') format('woff');
+src: local('Airbnb Cereal App Light'), url('./fonts/airbnb-cereal-app-cufonfonts-ttf/AirbnbCerealLight.tff') format('tff');
+}
+
+
+@font-face {
+font-family: 'Airbnb Cereal App Medium';
+font-style: normal;
+font-weight: normal;
+src: local('Airbnb Cereal App Medium'), url('./fonts/airbnb-cereal-app-cufonfonts-woff/AirbnbCerealMedium.woff') format('woff');
+}
+
+
+@font-face {
+font-family: 'Airbnb Cereal App Bold';
+font-style: normal;
+font-weight: normal;
+src: local('Airbnb Cereal App Bold'), url('./fonts/airbnb-cereal-app-cufonfonts-woff/AirbnbCerealBold.woff') format('woff');
+}
+
+
+@font-face {
+font-family: 'Airbnb Cereal App Extra Bold';
+font-style: normal;
+font-weight: normal;
+src: local('Airbnb Cereal App Extra Bold'), url('./fonts/airbnb-cereal-app-cufonfonts-woff/AirbnbCerealExtraBold.woff') format('woff');
+}
+
+
+@font-face {
+font-family: 'Airbnb Cereal App Black';
+font-style: normal;
+font-weight: normal;
+src: local('Airbnb Cereal App Black'), url('./fonts/airbnb-cereal-app-cufonfonts-woff/AirbnbCerealBlack.woff') format('woff');
+}
+
+.parent{
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  background-color: blue;
+  border: solid 3px black;
+  height: 295px;
+  width: 100%;
+  background-image: url('../../hero-placeholder.png');
+  background-position: left;
+  background-size: contain;
+  background-repeat: no-repeat;
+}
+
+.shareAndSaveContainer {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  margin-top: 3%;
+  padding-left: 77%;
+}
+
+.saveButton {
+  margin-left: 20px;
+}
+
+.share {
+
+}
```

Comment:
If this is unused, please remove and include only when necessary. Let's keep unused code out of production.